flake.nix

```diff
@@ -3,6 +3,7 @@
 
   inputs = {
     holonix.url = "github:holochain/holonix?ref=main";
+    holonix.inputs.holochain.url = "github:holochain/holochain?ref=fix/set-crypto-provider-for-tls-connections-in-holochain-binary";
```

Comment:
I guess we want to update this once the corresponding holochain PR has been merged?

Reply:
Absolutely.

Reply:
I would like to merge this anyway to carry on with Tryorama. This only affects the tests in the workflow, and it takes a little longer to release Holochain again.